test/compflags/ferguson/default-binary-name/test-default-binary-name3.chpl

```diff
@@ -0,0 +1,3 @@
+module M1 {
+  use M3;
+}
```

Comment:
Needs newline at end of file

Reply:
I don't know why you guys are so obsessed with spending precious disk space on final newlines... :)